### PR TITLE
DOCS-9279 clarify main file in root of zip file

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-9279 clarify main file in root of zip file

### CODE REVIEW

**Review by nicogon — DISMISSED** [review]
Lgtm

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 1]
```diff
@@ -96,6 +96,14 @@ To create an API asset:
 . (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, *Version*, and *API version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. You can change an API asset's version (asset version) and API version separately. The advanced settings are most often used to change the asset version.
 . Select *Publish*.
 
+To create a zip file containing all the files in a folder, with the main file in the root directory of the zip file, use a command like this, replacing `myfolder` with the name of your folder and `name.zip` with the name for the new zip file.
```
Comment (kbostrom) [inline modules/ROOT/pages/to-create-an-asset.adoc]: Per style ref., ZIP file
```suggestion
To create a ZIP file containing all the files in a folder, with the main file in the root directory of the ZIP file, use a command like this, replacing `myfolder` with the name of your folder and `name.zip` with the name for the new ZIP file:
```
Can you clarify ", with the main file in the root directory of the ZIP file,"? Do you mean that the ZIP file that you create contains all files in the folder but NOT the main file?

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 2]
```diff
@@ -96,6 +96,14 @@ To create an API asset:
...
+
+```
+cd myfolder; zip -r name.zip .
+```
```
Comment (kbostrom) [inline modules/ROOT/pages/to-create-an-asset.adoc:105]: Is that final space and period OK?

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 3]
```diff
@@ -96,6 +96,14 @@ To create an API asset:
...
+
+Do not use a command such as `zip -r name.zip myfolder`, which puts the folder in the root directory of the zip file, and causes an error message such as `The zip file does not contain a .raml file in the root directory`.
```
Comment (kbostrom) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
Do not use a command such as `zip -r name.zip myfolder`, which puts the folder in the root directory of the ZIP file and causes an error message such as `The zip file does not contain a .raml file in the root directory`.
```

Comment (kbostrom) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Of course this is out of context, but be sure that the audience will know the distinctions among the ZIP file, the folder files that compose it, the "main" file, the root directory,.... The hierarchy is difficult for me to imagine.

**Review by kbostrom — DISMISSED** [review]
CE per request

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-create-an-asset.adoc:105]: Yes, in a unix command, the "." separated by a space indicates the current directory.

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-create-an-asset.adoc]: You're right, that needs clarification. Rewrote and corrected "ZIP file".

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Good point. I have rewritten for clarity, so it should now match the audience's understanding.

**Review by Mulesoft-Mysti — APPROVED** [review]

**Pull request merged**